src/scrubbable/scrubbable.ts

```diff
@@ -15,7 +15,7 @@ type Scrubber = (node: DocumentFragment, editor: MinidocBase) => DocumentFragmen
 
 const isSafeUrl = (s?: string) => !s?.startsWith('javascript:');
 
-export const leafRules = { 'data-align': true };
+export const leafRules = { 'class': true };
```

Comment:
This means that we're going to end up with a lot of garbage class names when folks paste content in from external sources which is the primary reason I was trying to avoid this. I wonder if instead of a boolean here, we should add a whitelist: `{ class: ['text-right', 'text-center'] }` or something.

What do you think?